frontend/src/muralForm/MuralForm.tsx

```diff
@@ -15,58 +20,113 @@ const useStyles = makeStyles((theme: Theme) =>
       margin: theme.spacing(1),
       width: "195px",
     },
+    bottomButton: {
+      fontWeight: 500,
+    },
+    bottomButtonContainer: {
+      display: "flex",
+      justifyContent: "space-between",
+    },
   })
 );
 
 interface IMuralFormProps {}
 
 function MuralForm(props: IMuralFormProps) {
   const styles = useStyles();
-  const currYear = new Date().getFullYear();
+  const [name, setName] = useState<string>("");
+  const [year, setYear] = useState<number>(new Date().getFullYear());
   const [assistants, setAssistants] = useState<string[]>([]);
   const [socialMedia, setSocialMedia] = useState<string[]>([]);
-  //todo: delete these logs once we actually use the state
-  console.log(assistants)
-  console.log(socialMedia)
+  const [artist, setArtist] = useState<number | null>(null);
+  const [borough, setBorough] = useState<number | null>(null);
+  const [addressCoords, setAddressCoords] = useState<number[]>([]);
+  const [description, setDescription] = useState<string>("");
+  const [neighbourhood, setNeighbourhood] = useState<string>("");
+  const [address, setAddress] = useState<string>("");
 
-  function updateAssistants(newAssistants: string[]) {
-    setAssistants(newAssistants);
+  function submitForm() {
+    if (name === "") {
```

Comment:
these conditions could be combined into a `switch` statement unless you're morally opposed to them like Martin Robillard

Reply:
but on what expression would we switch? they all test different values